internal/filesystem/zz_overflow_test.go

```
@@ -0,0 +1,55 @@
+		t.Fatal(err)
+	}
+	h := &ovHandler{}
+	w.Add(dir, h)
+	stop := make(chan struct{})
+	w.Start(stop)
+	defer close(stop)
+	time.Sleep(1500 * time.Millisecond)
+
+	const burst = 2000
+	for i := 0; i < burst; i++ {
+		os.WriteFile(filepath.Join(dir, fmt.Sprintf("b-%04d.yaml", i)), []byte("x"), 0644)
```

golangci-lint (2.12.2)

[error] Error return value of `os.WriteFile` is not checked

(errcheck)